roles/ceph-mon/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- include: deploy_monitors.yml
```

Comment:
We have been using the style of including everything in main.yml unless the code takes a logical fork on a conditional:
```
include: foo.yml
when: foo.enabled 
```